### PR TITLE
feat(locale): Add TH (Thai) Translation

### PR DESCRIPTION
**Pull request by isudnop**
Hi Everyone

I would like to contribute TH translation for Turnip Prophet, since I see a lot of Thai players using this app a lot.

Screenshot:

Screen Shot 2563-04-28 at 18 24 43
Screen Shot 2563-04-28 at 18 24 49
Screen Shot 2563-04-28 at 18 24 55

Any comment and suggestion are welcome! :)

PS. I will ask my friend to check typo and gramma for Thai word and will confirm here again!

### CODE REVIEW

File: js/translations.js [hunk 1]
```diff
@@ -27,7 +27,8 @@ i18next
     ['pt-BR', 'Português'],
     ['ru', 'Русский'],
     ['zh-CN', '简体中文'],
-    ['zh-TW', '繁體中文']
+    ['zh-TW', '繁體中文'],
+    ['th', 'ไทย'],
```
Comment (kunpakorn2005) [inline js/translations.js:31]: Judge by other lang-code.
Maybe should update to icu-locale code as `th-TH`

but nice translation !

Reference: http://www.localeplanet.com/java/th-TH/index.html

Comment (theRTC204) [reply js/translations.js:31]: Echoing this - should be the full locale+language key name (it was recently discovered that many of the languages have the wrong key name 😓)

**Comment by isudnop** [comment]
@theRTC204 Okie, your comment and @kunpakorn2005 are addressed!

confirm with my friend that translation are all good, in the future if we can find more Thai friendly word for translation, I will open another PR

**Comment by mikebryant** [comment]
Thanks!

**Pull request merged**